features/supplier/opportunities.feature

```diff
@@ -75,3 +75,15 @@ Scenario: Checking all statuses returns all results
   And I check all 'status' checkboxes
   And I click the 'Filter' button
   Then I see that the stated number of results equals that result_count
+
+@withdrawn
+Scenario: See detail page for a withdrawn brief
+  Given I have a live digital outcomes and specialists framework
+    And I have a buyer
+    And I have a withdrawn digital-specialists brief
+  When I go to that brief page
```

Comment:
Let's follow the aligning of the cucumber documentation:
https://cucumber.io/docs/reference

(I know other files might do differently but I would suggest we write new ones in this style)